src/components/app/App.routing.tsx

```diff
@@ -19,11 +18,7 @@ const AppRouting: React.FC = () => {
           <TransitionGroup>
             <CSSTransition key={location.key} timeout={1000} classNames='page'>
               <Switch location={location}>
-                {[
-                  homeRoute,
-                  connectFourRoute,
-                  <Redirect key='redirect' to='/' />
-                ]}
+                {[connectFourRoute, <Redirect key='redirect' to='/' />]}
```

Comment:
if there's only one route or component to render, this should render the component itself

Reply:
this file could also be removed and the logic could be placed directly into the `App.component.tsx` file